src/Idea.js

```diff
@@ -1,8 +1,8 @@
 class Idea {
   constructor(title, body, id, star, image) {
-    this.id = id || Date.now();
     this.title = title;
     this.body = body;
+    this.id = id || Date.now();
```

Comment:
Good catch on this. I didn't notice that, but on looking back that's a necessary improvement.